lib/Command/SyncAccount.php

```diff
@@ -113,7 +114,8 @@ private function sync(Account $account, bool $force, OutputInterface $output): v
 				throw $e;
 			}
 
-			$output->writeln("Batch of new messages sync'ed");
+			$mbs = (int)(memory_get_usage() / 1024 / 1024);
```

Comment:
Maybe add a check that the function exists just in case this is blocked by a security policy

Reply:
Most disabled functions I've seen were at hosters and they don't give CLI access either. Therefore I think this will show up very rarely.